### PR TITLE
Rename task to dev:test_data:create

### PR DESCRIPTION
**Pull request by saraycp**
The task is renamed in [open-build-service](https://github.com/openSUSE/open-build-service/pull/13394).

### CODE REVIEW

**Comment by saraycp** [comment]
Do not merge until the [PR #13394](https://github.com/openSUSE/open-build-service/pull/13394) in open-build-service is merged.

**Review by dmarcoux — APPROVED** [review]

**Comment by dmarcoux** [comment]
> Do not merge until the [PR #13394](https://github.com/openSUSE/open-build-service/pull/13394) in open-build-service is merged.

Done before merging here.

**Pull request merged**
